src/styles/zuehlke-toc.css

```diff
@@ -1,88 +0,0 @@
-
```

Comment:
I just realised we need these styles, so they need to be updated too. Most courses use this as overview page linking to the chapters.

Reply:
Design-proposal to have it aligned with the new slide-deck:

![zuehlke-toc-slide](https://user-images.githubusercontent.com/36601005/189474744-36df99b5-3459-404d-b792-2181c9579f1c.jpg)


What do you think?

Reply:
Discussed offline, will be handled in a separate PR